cic_dev/addon/pre-requisites/remove-app

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+echo "Update Doeckerfile for remove apk -- START"
+
+# Note: remove Settings will cause the device can't enter launcher ui issue
+cd $AIC_WORK_DIR
+mkdir -p update/root
+#Add following content into update/Dockerfile
+cat << EOF | sudo tee -a update/Dockerfile
+#RUN ["/system/bin/sh", "-c", "rm -rf /system/priv-app/Settings"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/priv-app/Contacts"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/priv-app/MusicFX"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/priv-app/DocumentsUI"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Email"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Gallery2"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Calendar"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/DeskClock"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Music"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Browser2"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/ExactCalculator"]
+RUN ["/system/bin/sh", "-c", "rm -rf /system/app/QuickSearchBox"] 
+EOF
+
```

Comment:
Hi Sundar, 

According to Lenovo latest requirement, they want to keep following apps in the release image:
Settings, Gallery, Deskclock, Camera, Browser

Pls help to update remove apps content as follow in you PR_16.
#Add following content into update/Dockerfile
cat << EOF | sudo tee -a update/Dockerfile
RUN ["/system/bin/sh", "-c", "rm -rf /system/priv-app/Contacts"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/priv-app/MusicFX"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/priv-app/DocumentsUI"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Email"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Calendar"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/app/Music"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/app/ExactCalculator"]
RUN ["/system/bin/sh", "-c", "rm -rf /system/app/QuickSearchBox"] 
EOF

Reply:
As discussed offline, except camera, everything else is taken care.